helm/reana/values.yaml

```diff
@@ -69,13 +69,13 @@ components:
     environment:
       SHARED_VOLUME_PATH: /var/reana
   reana_workflow_engine_cwl:
-    image: reanahub/reana-workflow-engine-cwl:0.7.0
+    image: reanahub/reana-workflow-engine-cwl:0.7.1
   reana_workflow_engine_yadage:
-    image: reanahub/reana-workflow-engine-yadage:0.7.0
+    image: reanahub/reana-workflow-engine-yadage:0.7.1
   reana_workflow_engine_serial:
-    image: reanahub/reana-workflow-engine-serial:0.7.0
+    image: reanahub/reana-workflow-engine-serial:0.7.1
   reana_job_controller:
-    image: reanahub/reana-job-controller:0.7.0
+    image: reanahub/reana-job-controller:0.7.1
```

Comment:
I wonder why the [helm validation](https://github.com/reanahub/reana/pull/448/checks?check_run_id=1385541297) is not passing if [it passed in `maint-0.7`](https://github.com/reanahub/reana/pull/444/checks?check_run_id=1384326721).

Reply:
It points to the last line of this file, so probably the "# everyday at 3am" comment.